test/common_test/TestLibLoader.hh

```diff
@@ -66,5 +74,12 @@ namespace physics
     }
     return "";
   }
+
+  private: static std::string libToTest;
+};
 }
 }
+
+std::string gz::physics::TestLibLoader::libToTest = std::string("");
```

Comment:
Defining a static variable in a header can be [problematic](https://stackoverflow.com/questions/18860895/how-to-initialize-static-members-in-the-header). I recommend putting the variable in a private function and make it a singleton.

```
private: static std::string& TestLibLoader::LibToTest() 
{
  static std::string libToTest = "";
  return libToTest;
}
```

Reply:
how does https://github.com/gazebosim/gz-physics/pull/458/commits/18f950c48f4a5a3171ce026e54cc920e06523d6f  look?